src/layers/doubleSymLayer.jl

```diff
@@ -82,9 +82,9 @@ end
 
 function initTheta(this::DoubleSymLayer{T})  where {T<:Number}
     theta = [vec(initTheta(this.K));
-             T(0.1)*ones(T,size(this.Bin,2),1);
-             T(0.1)*ones(T,size(this.Bout,2),1);
-             initTheta(this.nLayer)];
+    T(0.1)*ones(T,size(this.Bin,2),1);
```

Comment:
Will break if `T<:Integer`

Not sure if that is possible at this point, but you could say `where {T<:AbstractFloat}` to be safe

Reply:
Do you think this would also be faster? We've been using `Number` throughout the package but in fact everything pretty much schould be floats.

Reply:
Shouldn't have any effect on performance. Now that we have made the method parametric, any further typing is really just limiting the input to make sure the function is used as intended.